Tasks/VsTest/Strings/resources.resjson/en-US/resources.resjson

```diff
@@ -2,13 +2,15 @@
   "loc.friendlyName": "Visual Studio Test",
   "loc.helpMarkDown": "[More Information](https://go.microsoft.com/fwlink/?LinkId=624539)",
   "loc.description": "Run tests with Visual Studio test runner",
-  "loc.instanceNameFormat": "Test Assemblies $(testAssembly)",
+  "loc.instanceNameFormat": "Test Assemblies",
```

Comment:
Issue with newline char in the task title. Build team will be fixing this bug, and we will need to update it once the fix is available